src/pages/IDResolutionPage/IDResolveRedirectionPage.tsx

```diff
@@ -0,0 +1,104 @@
+import { CSSProperties } from 'react';
```

Comment:
@bilalesi 

I'm having some troubles testing.
I have started fusion locally. I'm using delta dev endpoint and I am logged in as localuser.

I am trying to resolve this resource which (according to the metadata) is created by localuser:
https://dev.nise.bbp.epfl.ch/nexus/v1/resources/copies/hippocampus-hub/_/TestResource2

I then try to go to this url, but I see an empty page. (I have encoded the `@id` of the resource)

http://localhost:8000/resolve/https%3A%2F%2Fdev.nise.bbp.epfl.ch%2Fnexus%2Fv1%2Fresources%2Fcopies%2Fhippocampus-hub%2F_%2FTestResource2

![Screenshot from 2023-10-12 17-35-10](https://github.com/BlueBrain/nexus-web/assets/11242410/480e9083-bc05-4cba-bbf4-c77fbef99b6a)

What am I doing wrong?

Reply:
When I encode the `self` of the resource, I see an error. I was actually expecting to see the resolved resource
![Screenshot from 2023-10-12 17-37-30](https://github.com/BlueBrain/nexus-web/assets/11242410/e17590d8-031e-4535-8d03-4fce7de4c641)

Reply:
the`ID != self`, so will not work, delta resolve endpoint was built for IDs, we already have this functionality for self working

Reply:
I fixed the issue, I commented out the accept header, this is why you are seeing the empty page
In the fix, you will get this message: (we can refine the text later) due cors

<img width="574" alt="Screenshot 2023-10-12 at 19 31 15" src="https://github.com/BlueBrain/nexus-web/assets/2632473/63c9378c-f825-4462-9749-8a12acda3d92">